runner.py

```diff
@@ -56,6 +56,8 @@ def parse_config(config_file):
         config['loggly_token'] = parser.get('loggly_log', 'token')
 
     config['mongo_db'] = parser.get('mongodb', 'database')
+    config['mongo_host'] = parser.get('mongodb', 'host')
```

Comment:
this should have default values so people could still use the old mnemosyne config.